pageserver/src/http/routes.rs

```diff
@@ -8,16 +8,18 @@ use remote_storage::GenericRemoteStorage;
 use tokio_util::sync::CancellationToken;
 use tracing::*;
 
+use pageserver_api::models as api;
```

Comment:
```suggestion
use pageserver_api::models;
```

I'd find this less surprising, but I guess it's no biggie with all of the uses we currently need.